com.unity.render-pipelines.core/Editor/CoreEditorStyles.cs

```diff
@@ -27,9 +27,13 @@ public static class CoreEditorStyles
         /// <summary><see cref="Texture2D"/> 1x1 pixel with blue color</summary>
         public static readonly Texture2D blueTexture;
 
-        /// <summary> PaneOption icon </summary>
+        /// <summary> PaneOption icon for dark skin</summary>
         static readonly Texture2D paneOptionsIconDark;
+
+        /// <summary> PaneOption icon for light skin</summary>
```

Comment:
Not required as not public :) (can keep it)

Reply:
I was in the mood to document everything xD